### ISSUE TITLE
Holdout symbionts from training set

### ISSUE THREAD

**Issue by dustin-mullaney** [SrivatsanLab/organism_scale_modelling#7]
In addition to just not being good training data, these are a potentially excellent validation set for filling in required genes to support a small set of desired genes
